vendor/github.com/emirpasic/gods/.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+go:
+  - 1.1.x
+  - 1.2.x
+  - 1.3.x
+  - 1.4.x
+  - 1.5.x
+  - 1.6.x
+  - 1.7.x
+  - 1.8.x
+  - 1.9.x
+  - 1.10.x
+  - 1.11.x
```

Comment:
Why so many versions. Wouldn't be enough to use `1.10` and `1.11`?

Reply:
It's updated by `make deps-update`, nothing manual

Reply:
Do we really need to support so many versions?